java8/src/main/java/cucumber/runtime/java8/Java8HookDefinition.java

```diff
@@ -1,17 +1,17 @@
-package cucumber.runtime.java;
+package cucumber.runtime.java8;
+
+import static java.util.Arrays.asList;
```

Comment:
I've found that the most common ordering of import in the code base is static imports after the other imports, so I've added that ordering to the Eclipse configuration I use for Cucumber-JVM, but you seems to have another configuration.